pkg/ccl/backupccl/backup_planning.go

```diff
@@ -333,6 +333,7 @@ func resolveOptionsForBackupJobDescription(
 	}
 
 	var err error
+	// MB note: should we be using cloud.RedactKMSURI(uri) here instead?
```

Comment:
nit: if you want to stick your name/initials on it, may as well use the standard `TODO(<name>)` syntax

Reply:
Done